src/app/api/proxy/[...proxy]/route.ts

```diff
@@ -38,18 +38,9 @@ enum RpcMethods {
 }
 
 type RpcRequestParams = any[];
-type RpcRequest = {
-  rpcMethod: RpcMethods;
-  params: RpcRequestParams;
-  bitcoinDUrl: string;
-};
-
-type RpcResponse = {
-  result: any; // this eventually will be a genric type based on the rpcMethod
-};
 
-const rpcUser = process.env.NEXT_PUBLIC_BITCOIN_RPC_USER_NAME || "devnet";
-const rpcPassword = process.env.NEXT_PUBLIC_BITCOIN_RPC_PASSWORD || "devnet";
+const rpcUser = process.env.BITCOIN_RPC_USER_NAME || "devnet";
+const rpcPassword = process.env.BITCOIN_RPC_PASSWORD || "devnet";
```

Comment:
Given we have these with defaults duplicated in a couple places, doesn't it make sense to parse the server configuration (filling in defaults) only once and then export it to other modules?

Reply:
done